health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -452,6 +506,40 @@ const FacilityPopUp = ({ details, onClose, updateDetails }) => {
+                onClick={
+                  
+                  handleUnsaasignTrue
+                }
+              />,
+              <Button type={"button"} size={"large"} variation={"primary"} label={t("NO")} onClick={handleUnsaasignFalse} />,
+            ]}
+            sortFooterChildren={true}
+            onClose={() => {
+              setConfirmUnasignPopup(false);
+            }}
+          ></PopUp>}
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)